resources/java/IfStatement.java

```diff
@@ -0,0 +1,25 @@
+public class IfStatementsExample {
+
+    /**
+     * Example of a basic if statement.
+     *
+     * @param value The value to check.
+     */
+    public void exampleIfStatement(int value) {
+        if (value > 0) {
+            System.out.println(value + " is greater than 0");
+        }
+        if (value > 0) {
+            System.out.println(value + " is greater than 0");
+        } else {
+            System.out.println(value + " is less than or equal to 0");
+        }
+        if (value > 0) {
+            System.out.println(value + " is greater than 0");
+        } else if (value < 0) {
+            System.out.println(value + " is less than 0");
+        } else {
+            System.out.println(value + " is equal to 0");
+        }
```

Comment:
```suggestion
        if (value > 0) return;
        if (value > 0) {
            System.out.println(value + " is greater than 0");
        }
        if (value > 0 && value > 1 ||  value > 5) {
            System.out.println(value + " is greater than 0");
        } else {
            System.out.println(value + " is less than or equal to 0");
        }
        if (value > 0) {
            System.out.println(value + " is greater than 0");
        } else if (value < 0) {
            System.out.println(value + " is less than 0");
        } else {
            System.out.println(value + " is equal to 0");
        }
```

Reply:
As this is used to calculate MCC, one could arise the question on how to handle your new example here regarding this metric. As effectively, returning immediately when the value > 0 would also render the other three if-loops basically useless, as the conditions of all of them will always be wrong. So the paths through this function effectively shrink to 3 here.

That is just a side node, as I do not suggest making the metric consider such things. To do such a thing, we would have to include a dead code detection or similar compiler-stuff.

Reply:
It is just an example to demonstrate another type of if condition. Please adapt it.
The metric should not take care of dead code btw.

Reply:
Done